src/main/java/com/google/api/codegen/transformer/go/GoGapicSurfaceTransformer.java

```diff
@@ -247,9 +255,13 @@ static ModelTypeTable createTypeTable() {
 
   @VisibleForTesting
   List<StaticLangApiMethodView> generateApiMethods(
-      SurfaceTransformerContext context, List<Method> methods, List<String> pageStreamImports) {
+      SurfaceTransformerContext context,
+      List<Method> methods,
+      List<String> pageStreamImports,
```

Comment:
Maybe a more general map from enum to List<String>? I suspect there will be more of these.